### ISSUE TITLE
[BUG] Error: 500: no_route

### ISSUE THREAD

**Issue by ChristianChiarulli** [getAlby/lightning-browser-extension#3004]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Can't make payment to address, could use a more descriptive error.

### Screenshots [optional]

<img width="426" alt="Screenshot 2024-01-27 at 8 17 07 PM" src="https://github.com/getAlby/lightning-browser-extension/assets/29136904/a8865106-0d30-4211-ae1b-10b784cd1730">

### Steps To Reproduce

try to make a payment to an unsupported address

### Expected behavior

A clearer message as to why this doesn't work

### Alby information

Installed through firefox extension store

### Are you working on this?

None

**Comment by acarlton** [comment]
I was experiencing an issue with no routes, but it turns out to be a deeper issue with Alby browser extension not logging into Nostr between two difference computers. Have you double-checked that the site you're trying to use appears in the Connected Sites list in the Alby settings?

**Comment by ChristianChiarulli** [comment]
the person who I was sending the payment too said that their node doesn't have sufficient capacity (not sure what this means) but I'm really just hoping for a better error message/flow here.

**Comment by jpl-btc** [comment]
@ChristianChiarulli I agree that there is room for improvement in how errors are communicated. Could you share what kind of error message or flow you would prefer to see or expect? Do you have any specific ideas or examples in mind? Thank you!
